.github/workflows/master-e2e.yaml

```diff
@@ -257,6 +257,7 @@ jobs:
           SPEC: |
             /workdir/e2e/unit_tests/first_login_rancher.spec.ts
             /workdir/e2e/unit_tests/p0_fleet.spec.ts
+            /workdir/e2e/unit_tests/provisioning_gitrepo_tests_fleet.spec.ts
```

Comment:
after qa meeting agreed to be renamed to `p1_fleet.spec.ts`if considered p1

Reply:
Sure, I have this in my mind to update it.